augur/postprocess.py

```diff
@@ -74,192 +69,112 @@ def postprocess(config):
-    return vals, np.array(theta)
+    # Build a collection of the ellipses to display.
+    ellipses = EllipseCollection(
+        widths, heights, angles, units="xy", offsets=mu,
+        transOffset=ax.transData, facecolors=facecolors,
+        edgecolors=edgecolors, linestyles=linestyles, linewidth=linewidth)
+    ax.add_collection(ellipses)
+
+    sig0 = np.sqrt(C[0, 0])
+    sig1 = np.sqrt(C[1, 1])
+    return sig0, sig1
 
 
-def get_ellipse(fisher, par1, par2, inv_cache=None):
+def get_FoM_all(fisher, par1, par2):
     """
-    Routine to obtain the ellipse parameters
-    of a pair of parameters given a Fisher matrix.
-    This routine is similar to `get_all_fish` but slices for a given pair.
-    It may not give the same results as `get_all_fish` if the Fisher matrix is
-    not easily invertible.
+    Two alternative ways to get the FoM for a pair of parameters
+    (code stolen from https://github.com/CosmoLike/DESC_SRD/blob/master/fisher.py#L274)
 
     Parameters:
     -----
     fisher : ndarray
-        Array containing the Fisher matrix.
+        Fisher matrix
     par1 : int
-        Index of the first parameter to consider
-    par2: int
-        Index of the second paramter to consider
-    inv_cache:
-        Cached inverse Fisher matrix.
-    Returns:
-    -----
-    vals : ndarray
-        eigenvalues of the particular pair.
-    theta : float
-        degeneracy angle of the pair.
-    """
-    inv_fisher = np.zeros((2, 2))
-    if inv_cache is None:
-        inv_cache = np.linalg.inv(fisher)
-    inv_fisher[0, 0] = inv_cache[par1, par1]
-    inv_fisher[0, 1] = inv_cache[par1, par2]
-    inv_fisher[1, 0] = inv_cache[par2, par1]
-    inv_fisher[1, 1] = inv_cache[par2, par2]
-    fisher_red = np.linalg.inv(inv_fisher)
-    smallC = np.linalg.inv(fisher_red)
-
-    vals, vecs = eigsorted(smallC)
-    theta = np.degrees(np.arctan2(*vecs[:, 0][::-1]))
-    return vals, theta, inv_cache
-
-
-def plot_ellipse(vals, theta, color, label, nsigma, lw, ls, cent0, cent1, ax=None):
-    """
-    Routine to generate plots of a pair of Parameters
-
-    Parameters:
-    ------
-    vals : ndarray
-        Array of eigenvalues
-    theta : float
-        Angle of degeneracy (in degrees).
-    color : sting or matplotlib.color
-        Color of the line of the Ellipse object drawn.
-    label: string
-        Label describing the Ellipse to plot.
-    lw : float
-        Linewidth parameter from matplotlib.
-    ls : string
-        Linestyle parameter from matplotlib.
-    cent0 : float
-        x-coordinate of the centroid of the Ellipse.
-    cent1 : float
-        y-coordinate of the centroid of the Ellipse.
-    ax : matplotlib.pyplot.axis
-        Axis in which to plot the Ellipse
+        Index of first parameter
+    par2 : int
+        Index of second parameter
 
     Returns:
     -----
-    Ellipse : matplotlib.patches.Ellipse
-        Ellipse for the selected values
+    FOM : float
+        Estimate of FoM.
```

Comment:
Maybe we can clarify this is the one for the DETF, and that it corresponds to the 68% CL area.

Reply:
Yeah, so it's just the square root of the determinant which is the one that is used in the SRD (footnote 4 in page 6 of https://arxiv.org/pdf/1809.01669.pdf). But I think that it's better to do what you suggest below and have the correct scaling on the FoM as well.